README.md

```diff
@@ -71,7 +71,7 @@ the `ScheduledSparkApplication` CRD and the corresponding CRD controller.
 
 Spark Operator currently supports the following list of features:
 
-* Supports Spark 2.3 and up.
+* This branch specifically supports Spark 2.2 fork with Kubernetes Support (https://github.com/apache-spark-on-k8s/spark)
```

Comment:
lets add a comment here, that we will be moving to 2.4 when available and 2.4 will deprecate this branch

Reply:
done